.travis.yml

```diff
@@ -18,13 +16,15 @@ env:
     - secure: "J9RE3MbU26kmTg5y8WwMD0RnW99SxjJvNY4uFfdbDuPlAn6BV9BQuq1qaPYGksdYdJ8WmXhlRKIhjXjZIgvIEzDQ0msPbJ3dU7uCJFrZRVG3I/jbZR0EfcOiWCaDuQU2TDZw5bo4qanTr4nnZpnBVqj0erjytxUkyqIuRAR3xOg="
     # AWS_S3_BUCKET_NAME_STATIC
     - secure: "LY/cIlI4DW9JAYbPK3pztVptQoaLyLU8UX7s1k9+OmJ/jkfJozbsC9PS3VAIQqJVSE78oRjp2tJOkpg1CsOxI6PttplMzoJznNBbQ6G5iCITugV+o0wuCesM31FqMratOWalZjoDMCdv/qb624KOO2o+0XR5EpLUk+/iJmtdtaI="
+    # AWS_S3_MAX_AGE_SECONDS
+    - AWS_S3_MAX_AGE_SECONDS=3600
 matrix:
   fast_finish: true
 cache:
   directories:
     - $HOME/.pip-cache/
 install:
-  - travis_retry pip install $DJANGO coverage requests --download-cache $HOME/.pip-cache
+  - travis_retry pip install -r tests/requirements.txt --download-cache $HOME/.pip-cache
```

Comment:
For installable packages, I prefer to specify dependences in setup.py install_requires, keeping requirements.txt for actual applications. 

For testing requirements, you can add an `extras_require` section:

``` py
    extras_require = {
        "test":  [
            "coverage",
            "requests",
        ],
    },
```